src/main/scala/gitbucket/core/service/RepositoryService.scala

```diff
@@ -434,7 +434,7 @@ trait RepositoryService { self: AccountService =>
     val preferred = PluginRegistry().renderableExtensions.map(extension => s"${fileBaseName}.${extension}")
 
     def choiceTemplate(files: List[FileInfo]): Option[FileInfo] =
-      files.find(f => preferred.contains(f.name)).orElse(files.find(f => f.name.startsWith(fileBaseName)))
+      files.find(f => preferred.contains(f.name)).orElse(files.find(f => fileBaseName == FileUtil.trimExtension(f.name)))
```

Comment:
`preferred` is list of `ISSUE_TEMPALTE.md` and `ISSUE_TEMPLATE.markdown`.
And I think `ISSUE_TEMPLATE.TXT` should be treated as template if `.md` not found.
(But I'm not considering that case of `ISSUE_TEMPLATE` and `ISSUE_TEMPLATE.TXT` are exist both)